DuckDuckGo/NavigationBar/View/AddressBarTextField.swift

```diff
@@ -486,6 +488,21 @@ final class AddressBarTextField: NSTextField {
         }
     }
 
+    private func switchTo(_ tab: OpenTab) {
```

Comment:
Can we encapsulate this logic somewhere else? (standalone class maybe?) It is slightly confusing to have logic for switching tabs in `AddressBarTextField`